test/fixtures/correct.js

```diff
@@ -396,3 +396,8 @@ const ReactComponent = () => {
 };
 
 noop(ReactComponent);
+
+// `react/prefer-stateless-function`.
+const PreferStatelessFunction = () => null;
+
+noop(PreferStatelessFunction);
```

Comment:
What do you think about testing class components with methods, to make sure that this rule allows that?